src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/cessations/IndividualBoCessation.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.cessations;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class IndividualBoCessation extends BeneficialOwnerCessation {
+
+    @JsonProperty("personName")
+    private Object personName;
+
+    @JsonProperty("birthDate")
+    private String birthDate;
+
+    public IndividualBoCessation(String appointmentId,
+            String appointmentType,
```

Comment:
Don't need to have appointmentType in constructor arg list. Can set to 5009.

Reply:
fixed